### ISSUE TITLE
Floating point exception @ loss.backward()

### ISSUE THREAD

**Issue by AhmedCheikhRouhou** [lucidrains/performer-pytorch#29]
Hello, thank you for this rich library for pytorch users.
I'm implementing a seq2seq system for OCR and I tried to use your implementation for the decoder side of it.
the encoder is a CRNN based (CNN + BLSTM). i added the PerformerLM(..) wrapped with AutoregressiveWrapper(..)
The loss returned from the model seems to be working good :
tensor(4.2084, device='cuda:0', grad_fn=<NllLoss2DBackward>)
However, the loss.backword() raise this exception:
Floating point exception (core dumped)
Is there something wrong with my implementation since the encoder cannot be wrapped with the same Autoregressive function?
Many thanks
